pkg/config/sample-config.txt

```diff
@@ -0,0 +1,26 @@
+version = 31
+credential.accessKey = xxx
+credential.secretKey = yyy    ## SecretKey cannot be changed
+region = us-east-1
+browser = on
+worm = on
+cache.drives = 
```

Comment:
provide an example here how would the `cache.drives` would look like

Reply:
I'll provide either a default or an example for those configuration parameters in the sample config file. 

@harshavardhana,
The sample config file does not contain all possible configuration parameters.
Do you think we need to include them all in it?